Gruntfile.coffee

```diff
@@ -3,13 +3,6 @@ module.exports = ->
   @initConfig
     pkg: @file.readJSON 'package.json'
 
-    # Updating the package manifest files
-    noflo_manifest:
-      update:
-        files:
-          'component.json': ['graphs/*', 'components/*']
```

Comment:
Might want to keep grunt-noflo-manifest around in this iteration, given that this module still supports browsers (and hence needs to keep `component.json` up-to-date